### DISCUSSION TITLE
Possible false-positive

### DISCUSSION THREAD

**Discussion by tvdijen** [composer-unused/composer-unused#461] [Q&A] [answered]
Hi!

I'm new to this package, so maybe I'm doing something wrong, but I may have hit a bug.
Hopefully you can help me identify if it's a bug or misuse.

My project `simplesamlphp/simplesamlphp` has a hard [dependency](https://github.com/simplesamlphp/simplesamlphp/blob/master/composer.json#L69) on `simplesamlphp/assert`.
It is used in _many_ places around the project, for instance [here](https://github.com/simplesamlphp/simplesamlphp/blob/master/src/SimpleSAML/Session.php#L716), and [here](https://github.com/simplesamlphp/simplesamlphp/blob/master/src/SimpleSAML/Stats.php#L78.

The thing is, this action reports it as being [unused](https://github.com/simplesamlphp/simplesamlphp/actions/runs/4441263980/jobs/7796122928#step:10:11). What am I doing wrong?

Thanks!

**Comment by icanhazstring** [comment]
Hi. Welcome. First of, thanks for using composer-unused.

Let me take a look into it 🙂

**Reply by icanhazstring** [reply]
On first checks, this should be marked as used. Will convert this into a bug, and dig a little deeper :)

**Reply by tvdijen** [reply]
Thank you very much for your swift response!

**Reply by icanhazstring** [reply, marked as answer]
Good news. Found the issue.
Fix coming up in the next release :)

**Reply by tvdijen** [reply]
Interesting! I see I reduced your code complexity ;)
Thanks much!!!
